website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -892,6 +894,183 @@ curl \
 }
 ```
 
+## Show Data Retention Policy
+
+<EnterpriseAlert>
+This endpoint is exclusive to Terraform Enterprise, and not available in Terraform Cloud.
+</EnterpriseAlert>
+
+`GET /organizations/:organization_name/relationships/data-retention-policy`
+
+| Parameter            | Description                                                         |
+| ---------------------| --------------------------------------------------------------------|
+| `:organization_name` | The name of the organization to show the data retention policy for. |
+
+
+This endpoint shows the data retention policy set on the organization.
+When no data retention policy is set on the organization, the site wide data retention policy will be used by default. Read more about [organization data retention policies](/terraform/cloud-docs/users-teams-organizations/organizations#data-retention-policies).
```

Comment:
yes, good point on the link. I was using `cloud-docs` all over the place so that things would work in the Vercel preview, but I can start moving content over to ptfe-releases